### ISSUE TITLE
Text inputs become laggy on linux

### ISSUE THREAD

**Issue by johnnyasantoss** [sparrowwallet/sparrow#785]
Error on Linux while holding backspace in a text field (using Gnome 43.2 - Manjaro rolling with x11)
```
(Sparrow:118866): Gdk-WARNING **: 16:07:30.240: XSetErrorHandler() called with a GDK error trap pushed. Don't do that.
Xlib: sequence lost (0x20000 > 0x1033b) in reply type 0x2!
```

After that, UI becomes janky and is nearly impossible to type in any text field. I was able to reproduce a few times with holding backspace and ctrl + backspace.

**Comment by craigraw** [comment]
Seems like it could be a recent Xlib bug, other apps also exhibit this issue: https://gitlab.freedesktop.org/xorg/lib/libx11/-/issues/176

**Comment by johnnyasantoss** [comment]
Yeah, I see those logs with `alacritty` (terminal emulator) as well, but it didn't get laggy after ctrl + backspace.

Tried again today and I could not reproduce again with ctrl + backspace on sparrow. Will count as fixed 🤷🏽 thanks!
